schema/rdl_schema_0.1.json

```
@@ -151,17 +151,56 @@
             "event_set": {
               "type": "object",
               "required": [
+                "id",
                 "hazard_type",
                 "analysis_type"
               ],
               "properties": {
+                "id": {
+                  "title": "Event set identifier",
+                  "type": "string",
+                  "description": "The unique identifier for this event set.",
+                  "minLength": 1
+                },
                 "hazard_type": {
-                  "$ref": "#/$defs/common_hazard_type",
-                  "description": "Type of hazard modelled"
+                  "title": "Hazard type",
+                  "type": "string",
+                  "description": "The main type of hazard phenomena in the modelled scenario(s).",
+                  "codelist": "hazard_type.csv",
+                  "openCodelist": false
                 },
                 "analysis_type": {
-                  "$ref": "#/$defs/common_analysis_type",
-                  "description": "Type of analysis, probabilistic or deterministic"
+                  "title": "Analysis type",
+                  "type": "string",
+                  "description": "The type of analysis used by the hazard model: probabilistic (multiple potential events characterised by different intensity and occurrence frequency); deterministic (frequency-independent representation, such as mean, median or max); empirical (obtained from real event observations).",
+                  "codelist": "analysis_type.csv",
+                  "openCodelist": false
+                },
+                "frequency_distribution": {
+                  "title": "Frequency distribution",
+                  "type": "string",
+                  "description": "The frequency distribution assumed for the occurrence of events over a multi-year timeline.",
+                  "codelist": "frequency_distribution.csv",
+                  "openCodelist": false
+                },
+                "seasonality": {
+                  "title": "Seasonality distribution",
+                  "type": "string",
+                  "description": "The seasonality distribution assumed for the occurrence of events across a calendar year.",
+                  "codelist": "seasonality.csv",
+                  "openCodelist": false
```

Add `enum` and link to codelist reference.

```suggestion
                  "description": "The seasonality distribution assumed for the occurrence of events across a calendar year, from the [seasonality distribution codelist](https://rdl-standard.readthedocs.io/en/{{version}}/reference/codelists/#seasonality_distribution)",
                  "codelist": "seasonality.csv",
                  "openCodelist": false,
                  "enum": [
                    "uniform",
                    "userDefined"
                  ]
```